pypit/scienceimage.py

```diff
@@ -0,0 +1,602 @@
+# Module for the ScienceImage class
+from __future__ import absolute_import, division, print_function
+
+import inspect
+import numpy as np
+
+#from importlib import reload
+import datetime
+
+from astropy.time import Time
+
+from pypit import msgs
+from pypit import processimages
+from pypit import arspecobj
+from pypit.core import arprocimg
+from pypit.core import arskysub
+from pypit.core import arextract
+from pypit import artrace
+from pypit import ginga
+
+from pypit import ardebug as debugger
+
+# For out of PYPIT running
+if msgs._debug is None:
+    debug = debugger.init()
+    debug['develop'] = True
+    msgs.reset(debug=debug, verbosity=2)
+
+# Does not need to be global, but I prefer it
+frametype = 'science'
+
+
+class ScienceImage(processimages.ProcessImages):
```

Comment:
Yep - I like this much better. Very clean, and I can follow the logic.

Reply:
Thanks!